utils/exchange-bootstrap.sh

```diff
@@ -119,42 +135,53 @@ then
 		"https://api.github.com/repos/${EXCHANGE_ORG}/${REPO_NAME}/hooks"
 fi
 
-# CircleCI: follow the project
-echo "CircleCI: Following the project"
-curl -sS --fail -X POST "https://circleci.com/api/v1.1/project/github/${EXCHANGE_ORG}/${REPO_NAME}/follow?circle-token=${CIRCLECI_TOKEN}"
+# NO longer needed, this API request fails everytime we call it
+# # CircleCI: follow the project
+# echo "CircleCI: Following the project"
+# curl -v -sS --fail -X POST \
+#      --header "Circle-Token: ${CIRCLECI_TOKEN}" \
+#      "https://circleci.com/api/v1.1/project/github/${EXCHANGE_ORG}/${REPO_NAME}/follow"
+
 
 # CircleCI: upload the read-write key
 echo "CircleCI: Adding read-write SSH key"
 curl -sS --fail -X POST --header "Content-Type: application/json" \
+  --header "Circle-Token: ${CIRCLECI_TOKEN}" \
```

Comment:
inconsistent whitespace (tabs vs spaces strikes again)

Reply:
We're using 2 whitespaces across the st2 fleet for bash/script files and it indeed would be great following that in all the other places.

Mixing both (like this script) is indeed confusing.